homeassistant/util/unit_system.py

```diff
@@ -198,6 +202,7 @@ def as_dict(self) -> dict[str, str]:
     MASS_GRAMS,
     PRESSURE_PA,
     LENGTH_MILLIMETERS,
+    is_metric=True,
```

Comment:
This feels odd, why add this property?

The unit system itself defines "what" it is, not a property.

Should this be a class that implements a UnitSystem and thus uses inheritance?

Reply:
I want to take into account possible future `hybrid` system, which are neither fully `metric`, nor fully `US`, so I think having both properties explicit right away will help with that.

I'll see if I can a cleaner version using inheritance...

Reply:
Sure, SI could be added too (as an example), which would lead to another property `is_si` in the current format 😉